content/attacks/posts/2021-12-21-Exmo.md

```diff
@@ -0,0 +1,47 @@
+---
+date: 2021-12-21
+target-entities: EXMO
+entity-types:
+- Exchange
+- Custodian
+attack-types: Wallet Hack
+title: "EXMO Exchange Hot Wallets Compromised: Approximately $10.5 Million Stolen"
+---
+
+## Summary
+
+On December 21, 2021, the attackers gained unauthorized access to the EXMO exchange's hot wallets. Through these security weaknesses, they managed to execute malicious transactions, resulting in the theft of a substantial amount of cryptocurrency. 
```

Comment:
is this a different incident from this one: https://info.exmo.com/en/notifications/the-2020-exmo-security-incident-resolved/

I know you double checked the dates but just want to get it straight in my head.

Reply:
My bad, confused years